src/Form/Extension/ChoiceTypeExtension.php

```diff
@@ -27,6 +30,25 @@
  */
 final class ChoiceTypeExtension extends AbstractTypeExtension
 {
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        if ($options['multiple'] && (true === ($options['sortable'] ?? false))) {
+            $builder->addEventListener(FormEvents::PRE_SUBMIT, static function (FormEvent $event) use ($options) {
+                /** @var PreSubmitEvent $event */
+                $form = $event->getForm();
+                $data = $event->getData();
+
+                if (!is_array($data) || count($data) !== 1) {
+                    return;
+                }
+
+                if (str_contains($data[0], ',')) {
+                    $event->setData(explode(',', $data[0]));
```

Comment:
While it's work for your usage (with the javascript),
I assume it will introduce a bug if someone use this Extension/FormType in another situation for texte with string like
```
['Foo, bar']
```
It will become `['Foo', 'bar']` instead of staying the same...

I dunno if there is a way to avoid such thing...